webapp/channels/src/components/drafts/draft_row.tsx

```diff
@@ -245,7 +259,7 @@ function DraftRow({
         }
     }, [thread?.id]);
 
-    if (!channel) {
+    if (!channel && !isScheduledPost) {
```

Comment:
Along the same lines of reducing special handling for scheduled posts, could we actually get rid of this early return and let regular drafts render with a non-loaded channel as well? Regardless of if we render null or render a placeholder, we're still handling the case where we don't have a channel, so I think it could behave the same way for both drafts and scheduled posts

Reply:
It doesn't matter because we never show drafts for channels you are not member of, so I don't want to make unnecessary changes. There will always be some special handling of scheduled posts, we can't always avoid it.